server/src/test/java/com/broadcom/lsp/cobol/LangServerBootstrapTest.java

```diff
@@ -49,10 +65,12 @@ void initCtx() {
 
     // Bound class in Service module
     CobolLanguageServer server = getInjector().getInstance(CobolLanguageServer.class);
+    CustomThreadPoolExecutor customExecutor = LangServerCtx.getInjector().getInstance(CustomThreadPoolExecutor.class);
```

Comment:
Could you, please, rewrite it to avoid using the `LangServerCtx`? We plan to get rid of this class

Reply:
Ok, then   both server  and customExecutor  via constructor.  LangServerCtx is used in test, but can be removed.

Reply:
> Could you, please, rewrite it to avoid using the `LangServerCtx`? We plan to get rid of this class

As a part of  #621 this will be refactored